templates/config.json

```
@@ -587,6 +626,10 @@
       "provider": "JupiterOne",
       "summary": "Security and compliance roles and responsibilities are clearly defined to ensure segregation of duties.",
       "guidance": "Regulatory compliance frameworks, such as HIPAA, require formal assignment of security and privacy responsibilities to someone in your organization. As your organization expands and matures, you may want to form an executive level security committee to meet this requirement. Often, such a committee is a requirement of security certifications. You can manage this role assignment manually (I.E. in this documented policy/control procedure) or in the JupiterOne's app. Using JupiterOne, you can almost always avoid implementing a complicated enterprise GRC system.",
+      "instructionsSummary": "Confirm/tailor responsibilities.",
+      "instructions": [
+        "What responsibilities will you commit to?"
```

Similarly, we can combine the instructions summary and merge into the same string.

```suggestion
        "What responsibilities will you commit to? Confirm/tailor responsibilities."
```